src/geojson/geojson.ml

```diff
@@ -296,10 +299,11 @@ module Make (J : Geojson_intf.Json) = struct
               "A Geojson feature requires the type `Feature`. No type was \
                found.")
 
-    let to_json (t, props) =
+    let to_json (t, id, props) =
```

Comment:
Similarly here, the `to_json` function takes a `t` as an argument so you'll need to "destructure" the record rather than treat it as a tuple.

```suggestion
    let to_json { geometry; id; props } =
```